README.md

```diff
@@ -8,10 +8,16 @@ however we do not have enough for everyone. Beware of charge-only cables, they w
 programming your Circuit Playground Express! **Please remember to bring a _DATA CAPABLE_ micro USB
 cable with you!**
 
-**The Adafruit Circuit Playground Express (CPX) has CircuitPython on board! It’s a MicroChip
+We will be hosting Open Spaces every day at PyCon, May 3 - May 5, 2019. We'll have this Quickstart
+available as a worksheet, including some examples on the back to get you started. This repo will
+have those examples and more available for you to try, play with, and modify. We'll also have some
+peripherals (servos, potentiometers and external NeoPixel strips) available for you to connect to
+your Circuit Playground Express.
+
+**The Adafruit Circuit Playground Express (CPX) has CircuitPython on board!** It’s a MicroChip
```

Comment:
I could be very wrong, but I don't think the company name is camelCased. Their website has simply `Microchip` everywhere. Maybe it could be here as `Microchip Technologies™`?

Reply:
I'll fix the camelcase, but I think it's fine as `Microchip`.